thunder/core/jit_ext.py

```diff
@@ -915,6 +915,20 @@ def _general_jit_torch_autograd_function_apply_lookaside(obj: Any, *args, **kwar
     return _interpret_call(custom_forward, wrapped_ctx, *args_, **kwargs_)
 
 
+@general_jit_lookaside(torch.finfo)
+def _general_jit_torch_finfo_lookaside(obj: Any):
+    try:
+        thunder_dtype = unwrap(obj)
```

Comment:
It's nitpicky, but  would move the unwrap outside the try...except as an error there is an error in the interpreter itself.

Reply:
Oh sorry, that makes more sense. let me update that!